Update aws_ec2 documentation for the use_ssm_inventory option (#1586)
ansible-collections/amazon.aws: docs/docsite/rst/aws_ec2_guide.rst

``use_ssm_inventory``
---------------------

> ``use_ssm_inventory: True`` will allow to fetch EC2 instance information from the AWS Systems Manager (SSM) inventory service into hostvars. By leveraging the SSM inventory data, the ``use_ssm_inventory`` option provides additional details and attributes about the EC2 instances in your inventory.

Reviewer: ```suggestion
``use_ssm_inventory: True`` enables fetching EC2 instance information from the AWS Systems Manager (SSM) inventory service into hostvars. By leveraging the SSM inventory data, the ``use_ssm_inventory`` option provides additional details and attributes about the EC2 instances in your inventory.
```
